libraries/AP_Math/AP_Math.cpp

```diff
@@ -9,8 +9,6 @@ bool is_equal(const FloatOne v_1, const FloatTwo v_2)
     return fabsf(v_1 - v_2) < std::numeric_limits<decltype(v_1 - v_2)>::epsilon();
 }
 
-template bool is_equal<int>(const int v_1, const int v_2);
-template bool is_equal<short>(const short v_1, const short v_2);
```

Comment:
The intention behind it is that this function can deal with mixed types. 
After this change it will throw assert if you compare a float with an int.